pkg/minikube/image/image.go

```diff
@@ -321,22 +318,3 @@ func normalizeTagName(image string) string {
 	}
 	return base + ":" + tag
 }
-
-func fixRemoteImageName(ref name.Reference, imgName string) (name.Reference, string, error) {
```

Comment:
are u sure this is the right thing ? this was intially added by a chinnese user that verified it was correct, do u mind pasting the Output in the PR descrption, Before and After this PR so we know for sure this is the right thing to do ?

there is no way for my to verify it myself

Reply:
Hi @medyagh 

The image name is adapted in a more common place. https://github.com/kubernetes/minikube/pull/13782/files#diff-374e09b8c2cd5d8017b1c712fe5b41354849ac239ff1c75d027995a4594a3500

Reply:
I will verify this RP in the weekend